### PR TITLE
Fix bright inverse colors

### PR DESCRIPTION
**Pull request by DarkDefender**
Fixes https://github.com/jwilm/alacritty/issues/611

First time coding in Rust so I'm sure that this could potentially be solved in a better way...

### CODE REVIEW

File: src/term/mod.rs [hunk 1]
```diff
@@ -310,19 +310,27 @@ impl<'a> Iterator for RenderableCellsIter<'a> {
                 };
 
                 // `Color` fg, bg
-                let (fg, bg) = cell.colors(selected);
+                let (fg, bg) = cell.colors();
 
                 // `Rgb` fg, bg
-                let fg = self.compute_fg_rgb(fg, &cell);
-                let bg = self.compute_bg_rgb(bg);
+                let rgb_fg;
```
Comment (jwilm) [inline src/term/mod.rs]: These lines can be cleaned up a bit.

```rust
// Apply inversion and lookup RGB values
let (fg, bg) = if selected || cell.inverse() {
    (self.compute_bg_rgb(bg), self.compute_fg_rgb(fg, &cell))
} else {
    (self.compute_fg_rgb(fg, &cell), self.compute_bg_rgb(bg))
};
```

File: src/term/cell.rs [hunk 2]
```diff
@@ -89,12 +93,8 @@ impl Cell {
     /// Get foreground and background colors adjusted for INVERSE flag
     ///
     /// First color is the foreground, second color is the background.
-    pub fn colors(&self, force_invert: bool) -> (&Color, &Color) {
-        if self.flags.contains(INVERSE) || force_invert {
-            (&self.bg, &self.fg)
-        } else {
-            (&self.fg, &self.bg)
-        }
+    pub fn colors(&self) -> (&Color, &Color) {
```
Comment (jwilm) [inline src/term/cell.rs]: Seems like this method isn't doing much anymore; maybe it should just disappear.

**Comment by DarkDefender** [comment]
Made the suggested changes. I hope it looks good now.

**Comment by jwilm** [comment]
Thanks for the changes! Looks great; landed.

**Pull request merged**